test_runner/src/main/kotlin/ftl/cli/firebase/test/android/configuration/AndroidLocalesListCommand.kt

```diff
@@ -19,7 +19,7 @@ import java.nio.file.Paths
 )
 class AndroidLocalesListCommand : Runnable {
     override fun run() {
-        println(AndroidCatalog.localesAsTable(projectId = AndroidArgs.load(Paths.get(configPath)).project))
+        println(AndroidCatalog.localesAsTable(projectId = AndroidArgs.loadOrDefault(Paths.get(configPath)).project))
```

Comment:
I do not agree with this solution.
We should load config, 
if not present, we should read `System.getenv("GOOGLE_CLOUD_PROJECT") `and 
else we could load default config.

Do you agree with me?

Reply:
I think we loading in default config project_id from variable  
Check
```
CommonFlankConfig.default()
```
We use it on generating default configs
and in this function we have used: 
``` 
project = ArgsHelper.getDefaultProjectId() ?: ""

```
getDefaultProjectId() use 
```

ServiceOptions.getDefaultProjectId()

```

this function trying to load from System.getenv("GOOGLE_CLOUD_PROJECT")

Reply:
ok sorry I did not see it 👍